test/api/v3/unit/models/group.test.js

```diff
@@ -1173,11 +1173,12 @@ describe('Group Model', () => {
         it('updates participating members with rewards', async () => {
           await party.finishQuest(quest);
 
-          expect(User.update).to.be.calledOnce;
```

Comment:
Makes sense, thanks for the pointer! I've updated this to include tests with no failures, test with one failure followed by success, and then test for failing infinitely. Let me know if there are any other tests you think I should add.